src/pages/bulkImport/utils/flatfileValidators.js

```diff
@@ -46,40 +46,50 @@ export function validateMinMax(record) {
   return recordHookResponse;
 }
 
-export function validateAssetStrings(filenames, assetStringInputs) {
-  const validationMessages = assetStringInputs.map(
-    assetStringInput => {
-      const [assetString, rowIndex] = assetStringInput;
-      const assets = parseAssetString(assetString);
-      if (assets.length === 0) return null;
-      const [matchedAssets, unmatchedAssets] = partition(assets, a =>
-        filenames.includes(a),
-      );
+export function validateStrings(
+  validEntries,
+  stringInputs,
+  matchedMessage,
+  unMatchedMessage,
+  intl,
+) {
+  const validationMessages = stringInputs.map(stringInput => {
+    const [str, rowIndex] = stringInput;
+    const strs = parseBulkImportString(str);
```

Comment:
maybe something like `parsedString` instead of `strs`?

Reply:
Done.

Reply:
Change in https://github.com/WildMeOrg/codex-frontend/pull/431/commits/aeaa7cf1199b0121a3fcb6c48a39600a45ce0faf